_data/eileenxue/eileenxue.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Eileen Xue
+institution: Ryerson University
+profile_pic: eileenxue.jpg
```

Comment:
Git is case-sensitive. Could you please fix the file name? You could do one of the following:
1. Change `eileenxue.jpg` to `eileenxue.JPG` in the markdown file.
2. Rename `eileenxue.JPG` to `eileenxue.jpg`.
I'd suggest you rename the file!